config/vufind/EDS.ini

```diff
@@ -305,3 +305,19 @@ AU = None
 ;excludeLabel[] = 'Availability'
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
+
+; This section provides settings for optional caching of search requests. This
+; affects the primary Solr backend only; for caching of other backends, see the
```

Comment:
This comment mentions Solr and should probably be updated!

Reply:
Fixed.